modin/backends/base/query_compiler.py

```diff
@@ -33,11 +32,10 @@
 
 
 def _get_axis(axis):
-    def axis_getter(self):
-        ErrorMessage.default_to_pandas(f"DataFrame.get_axis({axis})")
-        return self.to_pandas().axes[axis]
-
-    return axis_getter
+    if axis == 0:
```

Comment:
Add docstring, please.

Reply:
#3021 adds docstring for this function, I think we shouldn't create conflicts

Reply:
Yes, I agree.